web-common/src/features/dashboards/filters/FilterChipsReadOnly.svelte

```diff
@@ -36,14 +41,18 @@ The main feature-set component for dashboard filters
 </script>
 
 <div class="relative flex flex-row flex-wrap gap-x-2 gap-y-2 items-center">
+  {#if timeRange}
+    <ReadOnlyTimeRange {timeRange} />
```

Comment:
It'd be nice symmetry with `DimensionFilterReadOnlyChip` and `MeasureFilterReadOnlyChip` if this component were named `TimeRangeReadOnlyChip`